### DISCUSSION TITLE
Not sure where to start regarding getting llama.cpp setup on Android pixel

### DISCUSSION THREAD

**Discussion by cloudliness** [ggml-org/llama.cpp#422] [Q&A]
You mention using cmake but coming from a non programming background, could anyone provide a step by step on how to go about adding the weights, setting up cmake and configuring termux? I've spent all day trying to get it set up and all I end up with is "permission denied". Thanks.

**Comment by shiipou** [comment]
Did you run the `termux-setup-storage` command ?
